src/theory/uf/theory_uf_rewriter.h

```diff
@@ -146,7 +148,7 @@ class TheoryUfRewriter {
     // cannot construct APPLY_UF if operator is partially applied or is not standard       
     return Node::null();
   }
-  // gets arguments, returns operator
+  // gets arguments, returns operator of a curried HO_APPLY node
   static Node decomposeHoApply(TNode n, std::vector<TNode>& args, bool opInArgs = false) {
```

Comment:
Just to make a bit clearer:
Given a (possibly nested) HO_APPLY node, it gathers all of its arguments in args, and then returns the operator.